crates/bevy_math/src/curve/easing.rs

```diff
@@ -324,20 +330,29 @@ mod easing_functions {
         }
     }
 
+    // These are copied from a high precision calculator; I'd rather show them
+    // with blatantly more digits than needed (since rust will round them to the
+    // nearest representable value anyway) rather than make it seem like the
+    // truncated value is somehow carefully chosen.
+    #[allow(clippy::excessive_precision)]
+    const LOG2_1023: f32 = 9.998590429745328646459226;
+    #[allow(clippy::excessive_precision)]
+    const FRAC_1_1023: f32 = 0.00097751710654936461388074291;
```

Comment:
I think having the more precise number in a comment is the best approach because overly precise number literals suggest that it can indeed be represented by `f32`, which is not the case. But with the comment pointing out there is a cutoff this is fine too.

Reply:
Id recommend putting the representable value as the constant, and the high-precision value as a comment. Ideally, include a procedure for generating these values too, for transparency.